Python/other/Minimize_the_Heights.py

```diff
@@ -0,0 +1,38 @@
+def minimize_the_heights(arr,k):
```

Comment:
Add a beginner friendly description

Reply:
@HarshCasper please check the as you told me to change

Reply:
Done @HarshCasper